saleor/discount/migrations/0067_orderdiscount_voucher_to_voucher_code_migration.py

```diff
@@ -0,0 +1,92 @@
+from django.db import migrations
+from django.db.models import Exists, OuterRef
+
+
+# For batch size 1000 with 10_000 per model OrderDiscount/Voucher/VoucherCode objects
+# OrderDiscount Migration took 2.39 seconds.
+# OrderDiscount Memory usage increased by 15.65 MiB.
+# OrderLineDiscount Migration took 2.48 seconds.
+# OrderLineDiscount Memory usage increased by 4.61 MiB.
+# CheckoutLineDiscount Migration took 2.26 seconds.
+# CheckoutLineDiscount Memory usage increased by 3.23 MiB.
+
+# For batch size 1000 with 100_000 per model OrderDiscount/Voucher/VoucherCode objects
+# OrderDiscount Migration took 28.68 seconds.
+# OrderDiscount Memory usage increased by 163.00 MiB.
+# OrderLineDiscount Migration took 27.98 seconds.
+# OrderLineDiscount Memory usage increased by 47.99 MiB.
+# CheckoutLineDiscount Migration took 28.12 seconds.
+# CheckoutLineDiscount Memory usage increased by 35.52 MiB.
+
+
+BATCH_SIZE = 1000
+
+
+def set_voucher_to_voucher_code_in_model(ModelName, apps, schema_editor):
```

Comment:
Also the part `set_voucher_to_voucher_code` in the function name is not adequate. Consider maybe something like `set_voucher_code_in_model` or `assign_voucher_code_in_model`

Reply:
changed to set_voucher_code_in_model